src/github/get-diff-from-commit.ts

```diff
@@ -14,15 +15,39 @@ export const getFileDiffFromGithub = async ({
   base,
   head
 }: DiffFilesType): Promise<ReturnTypeOfDiffFiles> => {
-  console.log(`fetching file diff for ${base} ${head}`)
+  const {exitCode, stdout, stderr} = await exec.getExecOutput(
+    'git',
+    [
+      'diff',
+      '--name-only',
+      '--ignore-submodules=all',
+      `--diff-filter=ACDMRTUX`,
+      `${base}..${head}`
+    ],
+    {
+      cwd: '.',
+      ignoreReturnCode: true,
+      silent: false
+    }
+  )
+
+  if (exitCode !== 0) {
+    throw new Error(
+      `Failed to get diff files between ${base}..${head} Exit code: ${exitCode}. Due to error ${stderr}`
+    )
+  }
+
+  const allFiles = stdout.split('\n').filter(Boolean)
+
```

Comment:
for this to work you need both commits to be fetched first
```suggestion
await exec.getExecOutput('git' , [`fetch origin ${base}`])
await exec.getExecOutput('git' , [`fetch origin ${head}`])
  const {exitCode, stdout, stderr} = await exec.getExecOutput(
    'git',
    [
      'diff',
      '--name-only',
      '--ignore-submodules=all',
      `--diff-filter=ACDMRTUX`,
      `${base}..${head}`
    ],
    {
      cwd: '.',
      ignoreReturnCode: true,
      silent: false
    }
  )

  if (exitCode !== 0) {
    throw new Error(
      `Failed to get diff files between ${base}..${head} Exit code: ${exitCode}. Due to error ${stderr}`
    )
  }

  const allFiles = stdout.split('\n').filter(Boolean)

```